src/platform/monitoring/external-services/ExternalServicesError.jsx

```diff
@@ -20,12 +21,17 @@ class ExternalServicesError extends React.Component {
       PropTypes.oneOf(Object.values(EXTERNAL_SERVICES)),
     ).isRequired,
     getBackendStatuses: PropTypes.func.isRequired,
+    onRender: PropTypes.func,
   };
 
   componentDidMount() {
+    const { onRender, statuses } = this.props;
```

Comment:
I don't know where the `statuses` prop is coming from, but it works in the `render()` function